modules/installing-ocp-agent-gather-log.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-gather-log_{context}"]
+= Gathering log data from a failed Agent-based installation
+
+Use the following procedure to gather log data about a failed Agent-based installation to provide for a support case.
```

Comment:
Just a question, does this procedure need any prereqs. Its not always necessary, but I wanted to ask to make sure.

Reply:
I don't think it does. 

I'm assuming that users who got this far have satisfied the prereqs to perform an install in the first place. And step 4a (Export the kubeconfig file to your environment by running the following command) allows you to use the oc CLI, otherwise I believe the procedure would require the common "access to the cluster as an administrator" prereq.

Reply:
Gotcha! Yeah prereqs are not always necessary, and I know sometimes they get missed. So I mainly just wanted to ask. 👍